tests/MongoDB.Bson.Tests/IO/JsonWriterTests.cs

```diff
@@ -511,6 +511,24 @@ public void TestJavaScriptWithScope()
 #pragma warning restore 618, 1062
         }
 
+        [Fact]
+        public void TestGuidWhenStandardGuidRepresentationShouldBeTakenFromStandardSubType()
```

Comment:
I think a more precise name is `TestUuidStandardWhenGuidRepresentationIsUnspecified`